docs/user_guide/HOWTO/Upgrading_SIMP.rst

```diff
@@ -6,7 +6,7 @@ HOWTO Upgrade SIMP
 .. WARNING::
```

Comment:
@trevor-vaughan - i deleted the file and put a "Coming Soon" placeholder in so we don't forget abut it later. Does that make sense?